README.tfdoc.hcl

```diff
@@ -0,0 +1,813 @@
+header {
+  image = "https://raw.githubusercontent.com/mineiros-io/brand/3bffd30e8bdbbde32c143e2650b2faa55f1df3ea/mineiros-primary-logo.svg"
+  url   = "https://mineiros.io/?ref=terraform-aws-ecs-service"
+
+  badge "build" {
+    image = "https://github.com/mineiros-io/terraform-aws-ecs-service/workflows/Tests/badge.svg"
+    url   = "https://github.com/mineiros-io/terraform-aws-ecs-service/actions"
+    text  = "Build Status"
+  }
+
+  badge "semver" {
+    image = "https://img.shields.io/github/v/tag/mineiros-io/terraform-aws-ecs-service.svg?label=latest&sort=semver"
+    url   = "https://github.com/mineiros-io/terraform-aws-ecs-service/releases"
+    text  = "GitHub tag (latest SemVer)"
+  }
+
+  badge "terraform" {
+    image = "https://img.shields.io/badge/Terraform-1.x-623CE4.svg?logo=terraform"
+    url   = "https://github.com/hashicorp/terraform/releases"
+    text  = "Terraform Version"
+  }
+
+  badge "tf-aws-provider" {
+    image = "https://img.shields.io/badge/AWS-3.5-F8991D.svg?logo=terraform"
+    url   = "https://github.com/terraform-providers/terraform-provider-aws/releases"
+    text  = "AWS Provider Version"
+  }
+
+  badge "slack" {
+    image = "https://img.shields.io/badge/slack-@mineiros--community-f32752.svg?logo=slack"
+    url   = "https://mineiros.io/slack"
+    text  = "Join Slack"
+  }
+}
+
+section {
+  title   = "terraform-aws-ecs-service"
+  toc     = true
+  content = <<-END
+    A [Terraform] module to create and manage
+    [Amazon ECS Services](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/ecs_services.html)
+    on [Amazon Web Services (AWS)][aws].
+
+    **_This module supports Terraform version 1
+    and is compatible with the Terraform AWS Provider version 3.5 to 4.0**
+
+    This module is part of our Infrastructure as Code (IaC) framework
+    that enables our users and customers to easily deploy and manage reusable,
+    secure, and production-grade cloud infrastructure.
+  END
+
+  section {
+    title   = "Module Features"
+    content = <<-END
+      This module implements the following Terraform resources
+
+      - `aws_ecs_service`
+      - `aws_ecs_task_definition`
+    END
+  }
+
+  section {
+    title   = "Getting Started"
+    content = <<-END
+      Most common usage of the module:
+
+      ```hcl
+      module "terraform-aws-ecs-service" {
+        source = "git@github.com:mineiros-io/terraform-aws-ecs-service.git?ref=v0.0.1"
```

Comment:
`cluster` and `task_definition` are missing here which are both variables that are required for a working example. For an example please see the existing tests

Reply:
Good catch, thanks!